src/main/java/ch/wisv/areafiftylan/controller/TicketTransferRestController.java

```diff
@@ -44,6 +50,15 @@ public TicketTransferRestController(TicketService ticketService) {
         return createResponseEntity(HttpStatus.OK, "Ticket transfer successfully cancelled");
     }
 
+    @PreAuthorize("isAuthenticated() and @currentUserServiceImpl.hasAnyTicket(principal)")
```

Comment:
I don't know why GH marks this as outdated. It isn't. I'll re-comment for visibility.